Dockerfile

```
@@ -0,0 +1,43 @@
+WORKDIR /src
+COPY . .
+
+RUN cmake -S . -B build -G Ninja \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DWITH_QT=OFF \
+    -DWITH_OPENCL=OFF \
+    -DWITH_PHASESHIFTS=OFF \
+    -DINSTALL_DOC=OFF \
+    -DBUILD_TESTING=OFF
+
+RUN cmake --build build --parallel
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)